README.md

```diff
@@ -1,22 +1,27 @@
 # Doctrine Docs Builder
 
 A package for generating Doctrine repo docs in a unified way. Its purpose is to abstract the underlying method away from
-the generating command. It assumes the input directory is in `docs/en` at the root of the repo (for now).
+the generating command. It assumes the input directory is `docs/en/` (for now). It uses `phpdocumentor` tooling for 
+parsing the RST files.
+
+This project is considered an internal tool and therefore, you shouldn't use this project in your application. This 
+repository doesn't provide any support, and doesn't guarantee backward compatibility. Any or the entire project can 
+change, or even disappear, at any moment without prior notice.
```

Comment:
LOL this feels ominous

Reply:
Don't shoot the messenger! I used [this example](https://github.com/symfony-tools/docs-builder/) @stof provided